src/Inkluzitron/Services/BirthdayNotificationService.cs

```diff
@@ -39,25 +39,40 @@ public class BirthdayNotificationService : IScheduledTaskHandler
             Logger = logger;
         }
 
+        static private bool HasBirthdayOnTwentyNinthOfFebruary(User user)
```

Comment:
Lepší pojmenování by bylo `HasBirthdayInLeapYear`

Reply:
Z tebou navrhovaného názvu bych čekal, že by metoda dělala totéž, co `DateTime.IsLeapYear`

Reply:
> Z tebou navrhovaného názvu bych čekal, že by metoda dělala totéž, co `DateTime.IsLeapYear`

Přesně tak, rozhodně by to nebylo lepší pojmenování